src/smartHandler.ts

```diff
@@ -212,14 +217,17 @@ export class SMARTHandler implements Authorization {
             }
         }
 
-        if (patientOrgs) {
-            const { hostname, resourceType, id } = patientOrgs;
-            references.add(`${hostname}/${resourceType}/${id}`);
-            if (hostname === fhirServiceBaseUrl) {
-                references.add(`${resourceType}/${id}`);
-            }
-            if (request.resourceType && request.resourceType === resourceType) {
-                ids.add(id);
+        if (patientOrgs && patientOrgs.length) {
```

Comment:
Again, patientOrgs should be an array.

Reply:
`patientOrgs ` is  has definition as `patientOrgs?: FhirResource[]` in ` interface UserIdentity`. So it's already an array. Thus only checking if it's empty or not.